test/fixtures/basic/labelsMultilineOverflowHidden.js

```diff
@@ -0,0 +1,32 @@
+export default {
+  tolerance: 0.0025,
+  config: {
+    type: 'treemap',
+    data: {
+      datasets: [{
+        label: 'Simple treemap',
+        data: [6, 6, 4, 3, 2, 2, 1],
+        backgroundColor: 'red',
+        labels: {
+          display: true,
+          overflow: 'hidden',
+          formatter: (ctx) => ('value is ' + ctx.raw.v + ',').repeat(6).split(',')
+        }
+      }]
+    },
+    options: {
+      layout: {
+        padding: {
+          bottom: 10
```

Comment:
Why do we need a layout padding in these fictures?
Also looks like we should align the labels to pixles (should look what datalabels does)

Reply:
because without that space, I don't know how to download the image without triggering the hovering. Do you know how to do that? Maybe removing backgroundColor?

Reply:
oh right, I usually set `events = []`, so it won't trigger on anything.

Reply:
If you agree, I'm going to submit a PR to remove all paddings and resetting events. Now in my TODO